prod-sample/api/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:latest
+FROM node:12-alpine
```

Comment:
Not related to this change, but we were going to check if one-app runs on 14 need to try it out

Reply:
That's a good idea!